sbin/common/config_init.sh

```diff
@@ -309,7 +309,12 @@ function configDefaults() {
   BUILD_CONFIG[FREETYPE_FONT_VERSION]="2.9.1"
   BUILD_CONFIG[FREETYPE_FONT_BUILD_TYPE_PARAM]=""
 
-  BUILD_CONFIG[MAKE_COMMAND_NAME]="make"
+  if [ "${BUILD_CONFIG[OS_KERNEL_NAME]}" == "aix" ] || [ "${BUILD_CONFIG[OS_KERNEL_NAME]}" == "SunOS" ]; then
```

Comment:
Are we not covering this AIX case elsewhere?  How is it currently working?

Reply:
AIX was hacked on the machines whereas I've now made it universal